package.json

```diff
@@ -16,14 +16,16 @@
     "generate-sitemap": "node sitemap-generator.js",
     "test": "jest",
     "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "analyze": "ANALYZE=true yarn build"
   },
   "jest-runner-eslint": {
     "cliOptions": {
       "ignorePath": "./.gitignore"
     }
   },
   "dependencies": {
+    "@next/bundle-analyzer": "^11.1.2",
```

Comment:
This can't be a devDependcy, right?

Reply:
No we cannot it needs to go with `dependencies`